### PR TITLE
Add leaderstat ordering info

### PR DESCRIPTION
**Pull request by Sharksie**
## Changes

Added information about the leaderstat ordering API.

## Checks

By submitting your pull request for review, you agree to the following:

- [x] This contribution was created in whole or in part by me, and I have the right to submit it under the terms of this repository's open source licenses.
- [x] I understand and agree that this contribution and a record of it are public, maintained indefinitely, and may be redistributed under the terms of this repository's open source licenses.
- [x] To the best of my knowledge, all proposed changes are accurate.

### CODE REVIEW

**Comment by aetter** [comment]
Hey @Sharksie, do you have a code sample handy for testing/demonstration? I'm not clear on the implementation details. When you say "inside" it, do you mean parented to it, like this?

```
local function leaderboardSetup(player)
	local leaderstats = Instance.new("Folder")
	leaderstats.Name = "leaderstats"
	leaderstats.Parent = player

	local gold = Instance.new("IntValue")
	gold.Name = "Gold"
	gold.Value = 0
	gold.Parent = leaderstats
	
	local IsPriority = Instance.new("BoolValue")
	IsPriority.Value = true
	IsPriority.Parent = gold
	
	local Priority = Instance.new("NumberValue")
	Priority.Value = 2
	Priority.Parent = gold
end
```

**Comment by Sharksie** [comment]
Updated to mention the true boolvalue. @aetter

**Review by aetter — DISMISSED** [review]

File: content/en-us/players/leaderboards.md [hunk 1]
```diff
@@ -116,6 +116,16 @@ end
...
 
+## Ordering Stats
+
+There are three ways to control the order of stats in a leaderboard:
+
+- Add the stats in the order that you want them to appear.
+- Add a child `Class.BoolValue` named `IsPrimary` to the stat. Set its value to true to place the stat first in the list.
+- Add a child `Class.NumberValue` named `Priority` to the stat. Then set its value to an integer. Higher priority values appear first. Stats without a priority have a default priority of 0.
+
+`IsPrimary` takes precedence over over any `Priority` values. If multiple stats have `IsPrimary` set to true, `Priority` determines their order.
```
Comment (aetter) [inline content/en-us/players/leaderboards.md:127]:
```suggestion
`IsPrimary` takes precedence over any `Priority` values. If multiple stats have `IsPrimary` set to true, `Priority` determines their order.
```

**Review by aetter — APPROVED** [review]

**Comment by aetter** [comment]
Thanks so much @Sharksie!

**Pull request merged**